src/main/java/com/github/aloomaio/androidsdk/aloomametrics/AloomaAPI.java

```diff
@@ -116,21 +116,44 @@
      */
     public static final String VERSION = AConfig.VERSION;
 
+    private static final String LOGTAG = "AloomaAPI.AloomaAPI";
+    private static final String APP_LINKS_LOGTAG = "AloomaAPI - App Links (OPTIONAL)";
```

Comment:
In the other PR I had to shorten this Logtag, Android Logger only supports up to 23 characters